Sources/MarathonCore/Package.swift

```diff
@@ -14,7 +14,7 @@ public struct Package {
 }
 
 extension Package: Equatable {
-    public static func ==(lhs: Package, rhs: Package) -> Bool {
+    public static func == (lhs: Package, rhs: Package) -> Bool {
```

Comment:
Is this rule disablable? It doesn't make sense to me as functions are always defined `foo()` without a space.